### PR TITLE
catimg: add page

### PR DESCRIPTION
**Pull request by CairnThePerson**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.7.0

### CODE REVIEW

File: pages/common/catimg.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# catimg
+
+> Image printing in the terminal.
+> See also: `pixterm` and `chafa`.
```
Comment (CleanMachine1) [inline pages/common/catimg.md:4]:
```suggestion
> See also: `pixterm`, `chafa`.
```

File: pages/common/catimg.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> More information: <https://github.com/posva/catimg>.
+
+- Print a JPEG, PNG, or GIF to the terminal:
+
+`catimg {{path/to/file}}`
+
+- Use doubled [r]esolution:
+
+`catimg -r 2 {{path/to/file}}`
```
Comment (CleanMachine1) [inline pages/common/catimg.md:13]:
```suggestion
`catimg -r {{2}} {{path/to/file}}`
```

File: pages/common/catimg.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+> Image printing in the terminal.
+> See also: `pixterm` and `chafa`.
+> More information: <https://github.com/posva/catimg>.
+
+- Print a JPEG, PNG, or GIF to the terminal:
+
+`catimg {{path/to/file}}`
+
+- Use doubled [r]esolution:
```
Comment (CleanMachine1) [inline pages/common/catimg.md:11]:
```suggestion
- Increase the resolution the image is printed by a specified multiplier:
```

File: pages/common/catimg.md (hunk 1 above)
Comment (CairnThePerson) [reply pages/common/catimg.md:4]: Thanks, I'll update this on my other PRs as well.

File: pages/common/catimg.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+`catimg -r 2 {{path/to/file}}`
+
+- Disable 24-bit color for better [t]erminal support:
+
+`catimg -t {{path/to/file}}`
+
+- Specify custom [w]idth or [H]eight:
+
+`catimg {{-w|-H}} {{custom}} {{path/to/file}}`
```
Comment (CleanMachine1) [inline pages/common/catimg.md:21]:
```suggestion
`catimg {{-w|-H}} {{1000}} {{path/to/file}}`
```

I don't know whether this is correct for what the command needs.

File: pages/common/catimg.md (hunk 2 above)
Comment (CairnThePerson) [reply pages/common/catimg.md:13]: "2" is the only value that matters. The only other option is "1", which is the default. So I'm basically just showing the only possible use of the `-r` flag.

File: pages/common/catimg.md (hunk 3 above)
Comment (CairnThePerson) [reply pages/common/catimg.md:11]: Yeah, again, it's only possible to set the `-r` flag to 2. It's basically like `-r 2` is the whole flag.

File: pages/common/catimg.md (hunk 4 above)
Comment (CairnThePerson) [reply pages/common/catimg.md:21]: What would improve the clarity here? Should I always set some sort of default value?

Comment (CleanMachine1) [reply pages/common/catimg.md:21]: Yes, we like to give an example parameter that can be given 

Like 

`sleep {{5}}`

is better than 

`sleep {{number}}`

File: pages/common/catimg.md (hunk 3 above)
Comment (CleanMachine1) [reply pages/common/catimg.md:11]:
```suggestion
- Double the resolution of an image: 
```

File: pages/common/catimg.md (hunk 4 above)
Comment (CairnThePerson) [reply pages/common/catimg.md:21]: Alright! A reasonable value would be something like 40, so I'll set it to that.

```suggestion
`catimg {{-w|-H}} {{40}} {{path/to/file}}`
```

Comment (CleanMachine1) [reply pages/common/catimg.md]: Sure

File: pages/common/catimg.md (hunk 3 above)
Comment (CairnThePerson) [reply pages/common/catimg.md]: Thank you!

File: pages/common/catimg.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+> Image printing in the terminal.
+> See also: `pixterm`, `chafa`.
+> More information: <https://github.com/posva/catimg>.
+
+- Print a JPEG, PNG, or GIF to the terminal:
+
+`catimg {{path/to/file}}`
+
+- Double the [r]esolution of an image:
```
Comment (CleanMachine1) [inline pages/common/catimg.md:11]:
```suggestion
- Double the [r]esolution of an image:
```

Comment (CairnThePerson) [reply pages/common/catimg.md]: Ah, oops. Beat you to it.

File: pages/common/catimg.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+`catimg -r 2 {{path/to/file}}`
+
+- Disable 24-bit color for better [t]erminal support:
+
+`catimg -t {{path/to/file}}`
+
+- Specify custom [w]idth or [H]eight:
```
Comment (CleanMachine1) [inline pages/common/catimg.md:19]:
```suggestion
- Specify a custom [w]idth or [H]eight:
```

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Thanks looks good to me after that last suggestion

**Review by kbdharun — APPROVED** [review]

**Pull request merged**